pyvista/trame/ui/vuetify2.py

```diff
@@ -274,10 +52,10 @@ def ui_controls(self, mode=None, default_server_rendering=True, v_show=None):
             The UI view mode. Options are:
 
             * ``'trame'``: Uses a view that can switch between client and server
-              rendering modes.
+            rendering modes.
```

Comment:
These should still all be indented to get it to render right in sphinx.

Reply:
Thank you. Fixed in [8bcd586](https://github.com/pyvista/pyvista/pull/4811/commits/8bcd586cdfdd9289eb73b2ef22e1032bf228405b). Let me know if there are any I missed.